### ISSUE TITLE
[ question ] Is there a command such as close_low_diskspace= 

### ISSUE THREAD

**Issue by refriedfood** [rakshasa/rtorrent#727]
That doesn't close the client but rather will stop starting newly added torrents?
I find that closing the client causes a ratio issue and was wondering if there was a command to simply pause all new torrent until more space is allocated for better file management.

Thank you.

**Comment by kannibalox** [comment]
http://pyrocore.readthedocs.io/en/latest/custom.html#only-start-items-that-you-have-disk-space-for

**Comment by chros73** [comment]
> That doesn't close the client but rather will stop starting newly added torrents?

That's correct and probably not just the new ones. But there's a bug in 0.9.6/0.13.6 that affects this behaviour. Which version do you use?
You need proper queue management (e.g. with pyrocore) or do what @kannibalox suggested.

**Comment by refriedfood** [comment]
@ chros73 What's correct?

I'm using 0.9.6/0.13.6 on ubuntu 17.10

**Comment by chros73** [comment]
I meant that your statement is correct, meaning you're right :) :
it doesn't close the client but rather will stop downloads.

**Comment by refriedfood** [comment]
@chros73 
The command: schedule = low_diskspace,5,60,close_low_diskspace=10240M 
Will stop all downloads or just _new_ downloads and keep the others active?

I really just want to be able to have any new downloads in the watch folder paused once I'm almost out of drive space (10GB) but not have any of the others stop, or the client close. I'm afraid I didn't quite understand the pyrocore link.

**Comment by chros73** [comment]
> Will stop all downloads or just new downloads and keep the others active?

I don't remember, but the problem is that this functionality is kind of broken in 0.9.6 due to this [bug](https://github.com/rakshasa/libtorrent/pull/108). The result is that rtorrent could be crashed, since your command (the check itself) is only triggered in every 1 minute (60 secs).

**Comment by SteveMalso** [comment]
schedule = low_diskspace,5,1,close_low_diskspace=60G

Works for me. Triggers every 1 second.

**Comment by pyroscope** [comment]
Use this… 

https://github.com/pyroscope/pyrocore/blob/bf148e241c9d7fe13e971a792a40bac588122161/src/pyrocore/data/config/rtorrent.d/00-default.rc#L19-L23


and add an additional disk space check in ``d.watch.start``.

Or try what happens if you do a conditional ``d.stop`` in an inserted_new handler, for *any* item (not just watchdir ones).
